capstone/src/main/webapp/index.html

```diff
@@ -21,12 +23,24 @@ <h1>INSERT TITLE HERE</h1>
           <h2>Top Positive Associations:</h2>
           <ul id="pos-associations">
             <li>Thing 1</li>
+<<<<<<< HEAD
+	    <li>Thing 2</li>
+	    <li>Thing 3</li>
             <li>Thing 2</li>
             <li>Thing 3</li>
           </ul>
           <h2>Top Negative Associations:</h2>
           <ul id="neg-associations">
             <li>Thing 1</li>
+	    <li>Thing 2</li>
+	    <li>Thing 3</li>
+          </ul>
+        </div>
+	<div class="stats-divider"></div>
+        <div id="second-stats" class="container second-stats">
+          <p>Something here about stats</p>
+        </div>
+	<div class="stats-divider"></div>
```

Comment:
Need to rebase with new master , this part has been deleted in the current master branch